documents/xaml-suggested-actions/xaml-designer-suggested-actions-extensibility-behaviors.md

```diff
@@ -0,0 +1,57 @@
+## Suggested Actions - Behaviors
+
+![extensibility-migration-architecture](xaml-suggested-actions-no-behaviors.png)
+
+![extensibility-migration-architecture](xaml-suggested-actions-behaviors.png)
+
+```cs
+public class ExampleBehaviorActionProvider : BehaviorActionProvider
+{
+    //Specify Behavior attached property name
+    public override string BehaviorPropertyName => "Triggers";
+
+    //Specify fully qualified type name of a class that implements a Behavior attached property
+    public override string BehaviorPropertyTypeFullName => "Microsoft.Xaml.Behaviors.Interaction";
+
+    public override string Header => "Microsotf Xaml Behaviors";
```

Comment:
Microsotf -> Microsoft
Also fix the image that shows the misspelling in xaml-designer-suggested-actions-extensibility.md